src/main/java/me/botsko/prism/database/sql/SQLIdMapQuery.java

```diff
@@ -145,45 +170,58 @@ public void findAllIdsPartial(String material, String stateLike, Consumer<List<I
             try (PreparedStatement st = conn.prepareStatement(query)) {
                 st.setString(1, material);
                 st.setString(2, stateLike);
-                try (ResultSet rs = st.executeQuery()) {
-                    List<IntPair> ids = new ArrayList<>();
-
-                    while (rs.next()) {
-                        ids.add(new IntPair(rs.getInt(1), rs.getInt(2)));
-                    }
-
-                    if (!ids.isEmpty())
-                        success.accept(ids);
-                    else
-                        failure.run();
-                }
+                handleIdResult(st,success,failure);
             }
         } catch (final SQLException e) {
             dataSource.getLog().error("Database connection error: ", e);
             e.printStackTrace();
         }
     }
 
-    public void map(String material, String state, int block_id, int block_subid) {
+    private void handleIdResult(PreparedStatement st, Consumer<List<IntPair>> success, Runnable failure) throws SQLException{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [WhitespaceAround: '{' is not preceded with whitespace.](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5438808)